guide/inheritance-wallet/succession.md

```diff
@@ -0,0 +1,229 @@
+
+### Timelock constraints
+
+After activating the wallet on their signing devices, Christina and David have successfully gained access to their inheritance. However, they cannot immediately spend the funds using their inheritance keys due to the timelock constraints set up by their parents.
+
+As we have covered on the [use case page]({{ '/guide/inheritance-wallet/overview/' | relative_url }}), the lock period for the inheritance key set is set to 12 months (6 months for the recovery path + 6 months for the inheritance key set). Since Alice and Bob's last transaction was 5 months ago, Christina and David will have to wait an additional 7 months before they can use their inheritance keys.
+
+{% include tip/open.html color="blue" icon="info" label="Faster access with primary keys" %}
+
+If Christina and David had access to their parents' signing devices, they could potentially use the recovery path after 6 months. However, they don't have this access, so they must wait for the full 12-month period. 
+
+{% include tip/close.html %} 
+
+### Distributing the inheritance
+
+After the 12-month waiting period, the inheritance key set becomes active. Christina and David meet to create the transactions that will distribute their inheritance. 
+
+Christina goes on to create the first transaction, which will send half of the funds to her own wallet.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-create %}
+
+The transaction has been created and is ready to be signed. Christina goes ahead and sings it with her hardware signing device.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-christina %} 
+
+After Christina is done, she shares the partially signed transaction with David by generating a QR code. David imports the transaction by scanning the code. After reviewing the details he adds his signature and the app broadcasts the fully signed transaction to the bitcoin network.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-david %} 
+
+While waiting for the first transaction to be confirmed, Christina and David repeat the process to create and sign a second transaction, sending the remaining 50% of the funds to David's personal wallet.
+
+With both transactions confirmed on the blockchain, the inheritance is now fully distributed. Christina and David each control their respective share of the funds in their personal wallets, completing the inheritance process as their parents had intended. This marks the successful transfer of assets and the fulfillment of Bob and Alice's inheritance plan.
+
+## Conclusion
+
+The inheritance wallet design explored in this reference implementation leverages multi-key wallets and relative timelocks to create a flexible and secure inheritance solution. While this approach offers several advantages, it also comes with some potential drawbacks. Let's examine the pros and cons:
+
+### Pros
```

Comment:
A unique point about bitcoin inheritance is that it cannot be overruled by the law. With traditional methods, the descendants could get a court order to get to the funds. But that's simply not possible with bitcoin (this is the "ruling from the grave" aspect).